src/rpc/wallet_rpc_api.py

```diff
@@ -72,10 +71,31 @@ def get_routes(self) -> Dict[str, Callable]:
             "/get_trade": self.get_trade,
             "/get_all_trades": self.get_all_trades,
             "/cancel_trade": self.cancel_trade,
-            "/create_backup": self.create_backup,
-            "/get_backup_info": self.get_backup_info,
+            "/rl_set_user_info": self.rl_set_user_info,
+            "/send_clawback_transaction:": self.send_clawback_transaction,
         }
 
+    async def rl_set_user_info(self, request):
+        wallet_id = uint32(int(request["wallet_id"]))
+        rl_user = self.service.wallet_state_manager.wallets[wallet_id]
+        origin = request["origin"]
+        try:
```

Comment:
Unnecessary try/except. All the stuff in rpc api is already inside try/except.
Use specific try/except when you want to return non generic error message. (str(e) is default already iirc)